main.go

```diff
@@ -21,30 +21,35 @@ import (
 	"fmt"
 	"os"
 	"strconv"
-	"time"
 
 	configv1 "github.com/openshift/api/config/v1"
-	machinev1 "github.com/openshift/cluster-api/pkg/apis/machine/v1beta1"
 	"github.com/openshift/cluster-machine-approver/pkg/controller"
 	"github.com/openshift/cluster-machine-approver/pkg/metrics"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog/v2"
 	control "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	ctrl "sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
-
-	"k8s.io/klog/v2"
 )
 
 func main() {
 	var cliConfig string
+	var APIGroup string
+	var managementKubeConfigPath string
+	var workloadKubeConfigPath string
 
 	flagSet := flag.NewFlagSet("cluster-machine-approver", flag.ExitOnError)
 
 	klog.InitFlags(flagSet)
 	flagSet.StringVar(&cliConfig, "config", "", "CLI config")
```

Comment:
@alexander-demichev can we please validate accepted apigroups and fail otherwise?